res_syntax/src/res_core.ml

```diff
@@ -1566,25 +1566,29 @@ and parseParameter p =
             let pat =
               let pat = Ast_helper.Pat.var ~loc (Location.mkloc lblName loc) in
               let loc = mkLoc startPos p.prevEndPos in
-              Ast_helper.Pat.constraint_ ~attrs:[propLocAttr] ~loc pat typ
+              Ast_helper.Pat.constraint_ ~attrs:(propLocAttr :: attrs) ~loc pat
+                typ
             in
             (attrs, Asttypes.Labelled lblName, pat)
```

Comment:
Should this be `[]`?
Do we have a test for this case?

Reply:
Good catch, I missed when I moved from syntax.